src/utils/clickup.ts

```diff
@@ -207,56 +207,34 @@ export class Clickup implements ReleaseStrategy {
         };
     }
 
-    async fetchTasksFromReleaseTagTask(release_tag_task_url: string): Promise<Issue[]> {
+    async fetchTasksFromReleaseTagTask(task_id: string, team_id: string): Promise<Issue[]> {
         const issues: Issue[] = [];
-
-        const { task_id, team_id } = this.getTaskIdAndTeamIdFromUrl(release_tag_task_url);
-
         const task = await this.http.get<Task>(`task/${task_id}?team_id=${team_id}&custom_task_ids=true`);
         this.regession_task = task;
         const { custom_fields } = task;
         const task_ids = this.getTasksIdsFromCustomFields(custom_fields);
-        for (const task_id of task_ids) {
-            const task = await this.fetchIssue(task_id);
-            issues.push(task);
-        }
+        const fetch_issues_promises = task_ids.map(task_id => this.fetchIssue(task_id));
+        const fetched_issues = await Promise.allSettled(fetch_issues_promises);
+        fetched_issues.forEach(fetched_issue => {
+            if (fetched_issue.status === 'fulfilled') {
+                issues.push(fetched_issue.value);
+            }
+        });
```

Comment:
```suggestion
        const fetch_issues_promises = task_ids.map(task_id => this.fetchIssue(task_id));
        const fetched_issues = await Promise.allSettled(fetch_issues_promises);
        return fetched_issues.map(fetched_issue => {
            if (fetched_issue.status === 'fulfilled') {
                return fetched_issue.value;
            }
        });
```
You can quick return the values via a map than looping and pushing to an array. This way you can remove the declaration above:
`const issues: Issue[] = [];`

and the return below:
`return issues;`